orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -293,14 +313,15 @@ def AnonymiseCallback(dataset):
     orthanc.LogInfo("Removed overlays")
 
     # Apply anonymisation.
-    with Path("/etc/orthanc/tag-operations.yaml").open() as file:
-        # Load tag operations scheme from YAML.
-        tags = yaml.safe_load(file)
-        # Apply scheme to instance
-        dataset = pixl_dcmd.apply_tag_scheme(dataset, tags)
-        # Apply whitelist
-        dataset = pixl_dcmd.enforce_whitelist(dataset, tags)
-        orthanc.LogInfo("DICOM tag anonymisation applied")
+    for tag_operation_file in project_config.tag_operation_files:
```

Comment:
I don't think iterating through the operations is what we want here, because each step has a whitelist approach which will mean it'll only keep the tags defined in all of your config files. Instead I think we should read all the yaml files, and merge them together, overwriting in the order of definition (and document this)

Reply:
I've thought about this more and have a different take.  I think for now let's just ensure that the list of configs is only a length of one. That way when we do have a usecase for overriding, we can use that but at the moment its a nice to have and doesn't provide value (because none of the projects we know about are using it)

Reply:
yeah tbh I find that pretty confusing so imagine whoever writes that config will too. Perhaps "base" is worth keeping only as a template and then each project has its own config? And then we just keep it as a single config.

Reply:
Or do something like:

```suggestion
    all_tags = {}
    # Combine tags
    for tag_operation_file in project_config.tag_operation_files:
        with tag_operation_file.open() as file:
            # Load tag operations scheme from YAML.
            tags = yaml.safe_load(file)
            all_tags.update(tags)

    # Apply scheme to instance
    dataset = pixl_dcmd.apply_tag_scheme(dataset, all_tags)
    # Apply whitelist
    dataset = pixl_dcmd.enforce_whitelist(dataset, all_tags)
    orthanc.LogInfo(f"DICOM tag anonymisation applied according to {tag_operation_file}")

    orthanc.LogWarning("DICOM tag anonymisation applied")
```